acquia/acquia_cms
# ACMS-753: Verify atomic nature of all acquia cms modules as part of profile refactor

**Motivation**

Fixes #[ACMS-753](https://backlog.acquia.com/browse/ACMS-753)

**Proposed changes**

As we have to follow modular approach, where each acms module can be enable/disabled individually. Hence we have move configuration from profile install directory and leave at module level, so that if any module gets enable it will automatically enable its related configurations.

**Testing steps**

In order to test this, we need to follow multiple  site install for each module individually and make sure site after install works fine and that module feature is also available.
 
**Merge requirements**
- [_Enhancement_] _Major change_, _Minor change_, _Bug_, _Enhancement_, and/or _Chore_ label applied
